lib/tasks/mutant.rake

```diff
@@ -69,11 +69,16 @@ end
 # Only include in this collection the models that matter and have specs.
 #
 def models
+  # TODO: Add Person model when feature flag SPLIT_ADDRESS has been removed
+  # had to remove to bypass the following error
+  # def split_address?
+  # -  (c100_application.version >= 3) || ENV.key?("SPLIT_ADDRESS")
```

Comment:
Mutant is complaining that if you remove the `ENV.key?("SPLIT_ADDRESS")` part, the tests continue to pass.
This is usually a smell that some of the code has not been tested properly.

As `Person` now contains more code, some of it quite important, I think we should not bypass it, and try to improve the tests so everything is covered and mutant is "happy".